apps/emqx_plugins/test/emqx_plugins_SUITE.erl

```diff
@@ -882,6 +882,173 @@ group_t_cluster_leave(Config) ->
     ),
     ok.
 
+group_t_cluster_force_sync_vsn({init, Config}) ->
+    OldInstallDir = filename:join(emqx_cth_suite:work_dir(?FUNCTION_NAME, Config), old),
+    ok = filelib:ensure_path(OldInstallDir),
+    NewInstallDir = filename:join(emqx_cth_suite:work_dir(?FUNCTION_NAME, Config), new),
+    ok = filelib:ensure_path(NewInstallDir),
+    #{package := OldPackage, release_name := OldPluginName} =
+        get_demo_plugin_package(#{
+            release_name => ?EMQX_PLUGIN_TEMPLATE_RELEASE_NAME,
+            git_url => ?EMQX_PLUGIN_TEMPLATE_URL,
+            vsn => "5.1.0",
+            tag => "5.1.0",
+            shdir => OldInstallDir
+        }),
+
+    #{package := NewPackage, release_name := NewPluginName} =
+        get_demo_plugin_package(#{
+            release_name => ?EMQX_PLUGIN_TEMPLATE_RELEASE_NAME,
+            git_url => ?EMQX_PLUGIN_TEMPLATE_URL,
+            vsn => "5.9.0-beta.1",
```

Comment:
```suggestion
            vsn => "5.9.0-beta.3",
```

Reply:
it's ok to keep beta.1 here.
cluster sync and purge should work with any two different versions.